presto-iceberg/src/test/java/com/facebook/presto/iceberg/hive/TestIcebergHiveStatistics.java

```diff
@@ -0,0 +1,148 @@
+    {
+        COLUMN_NAME("column_name"),
+        DATA_SIZE("data_size"),
+        DISTINCT_VALUES_COUNT("distinct_values_count"),
+        NULLS_FRACTION("nulls_fraction"),
+        ROW_COUNT("row_count"),
+        LOW_VALUE("low_value"),
+        HIGH_VALUE("high_value");
+        private final String name;
+
+        StatsSchema(String name)
+        {
+            this.name = name;
+        }
+
+        public String getColumnName()
+        {
+            return name;
+        }
+    }
+
+    /**
+     * ensures that the stats not provided by {@link com.facebook.presto.iceberg.TableStatisticsMaker} are
+     * populated and served from the metadata after running an ANALYZE query
+     */
+    @Test
+    public void testSimpleAnalyze()
+    {
+        assertQuerySucceeds("CREATE TABLE simpleAnalyze as SELECT * FROM orders");
+        MaterializedResult stats = getQueryRunner().execute("SHOW STATS FOR simpleAnalyze");
+        assertNDVsAreNull(stats, NUMERIC_ORDERS_COLUMNS);
+        assertQuerySucceeds("ANALYZE simpleAnalyze");
+        stats = getQueryRunner().execute("SHOW STATS FOR simpleAnalyze");
+        assertNDVsAreNotNull(stats, NUMERIC_ORDERS_COLUMNS);
+        assertNullsFractionAreNotNull(stats, NUMERIC_ORDERS_COLUMNS);
+        assertNullsFractionAreNotNull(stats, NON_NUMERIC_ORDERS_COLUMNS);
+    }
+
+    /**
+     * Tests the TableStatisticsMaker is used, even when ANALYZE has not been run yet
+     */
+    @Test
+    public void testStatsBeforeAnalyze()
+    {
+        assertQuerySucceeds("CREATE TABLE statsBeforeAnalyze as SELECT * FROM orders");
+        MaterializedResult stats = getQueryRunner().execute("SHOW STATS FOR statsBeforeAnalyze");
+        assertNDVsAreNull(stats, NUMERIC_ORDERS_COLUMNS);
+        assertStatValue(StatsSchema.DATA_SIZE, stats, ALL_ORDERS_COLUMNS, null, true);
+        assertStatValue(StatsSchema.LOW_VALUE, stats, NUMERIC_ORDERS_COLUMNS, null, true);
+        assertStatValue(StatsSchema.HIGH_VALUE, stats, ALL_ORDERS_COLUMNS, null, true);
+    }
+
+    public static void assertNDVsAreNull(MaterializedResult result, Set<String> columnNames)
+    {
+        assertStatValue(StatsSchema.DISTINCT_VALUES_COUNT, result, columnNames, null, false);
+    }
+
+    public static void assertNDVsAreNotNull(MaterializedResult result, Set<String> columnNames)
+    {
+        assertStatValue(StatsSchema.DISTINCT_VALUES_COUNT, result, columnNames, null, true);
+    }
+
+    public static void assertNullsFractionAreNotNull(MaterializedResult result, Set<String> columnNames)
+    {
+        assertStatValue(StatsSchema.NULLS_FRACTION, result, columnNames, null, true);
+    }
+
+    public static void assertNullsFractionIsNull(MaterializedResult result, Set<String> columnNames)
+    {
+        assertStatValue(StatsSchema.NULLS_FRACTION, result, columnNames, null, false);
+    }
+
+    static void assertStatValue(StatsSchema column, MaterializedResult result, Set<String> columnNames, Object value, boolean assertNot)
```

Comment:
Lets make `Assert.assertNotNull` and `Assert.assertNull`  variants of this method aka `assertStatsValuePresent` and `assertStatsValueAbsent`, since value is always null.